setup.py

```diff
@@ -76,14 +76,16 @@
 
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
-    'jupyter_client',
-    'ipython',
-    'ipykernel', # bless IPython kernel for now
+    'jupyter_client<6.0.0',
+    'ipython<6.0.0',
+    'ipykernel<5.0.0', # bless IPython kernel for now
```

Comment:
These shouldn’t be pinned down. requires_python takes care of this.

Reply:
What versions of python should be compatible with this release? Technically, 5.x still works on the newer versions.

Reply:
5.x should work with Python 2.7, ≥3.3. But there shouldn't be any need to pin these packages. Did you encounter a compatibility issue here? What prompted the pinning?

Reply:
(woops, pressed the wrong button)
jupyter_console requires promt_toolkit<2.0.0 but ipython requires a newer version, thus the pinning.
With the current setup.py I am able to install jupyter_console in python3.3+ so pinning the python version seems to me like limiting eligible targets for no good reason.

Reply:
ok, then let's leave ipython pinned as a known compatibility issue, but not ipykernel or jupyter_client if there's no reason there.